--- lilypond-2.22.2/aclocal.m4.orig	2022-03-18 16:13:32.567418463 +0100
+++ lilypond-2.22.2/aclocal.m4	2022-03-18 16:21:08.664947571 +0100
@@ -505,7 +505,7 @@ AC_DEFUN(STEPMAKE_GUILE, [
     AC_MSG_CHECKING([for guile])
     guile="guile"
     found="no"
-    for r in $GUILE guile guile2 guile2.2 guile2.0 guile-2.0 \
+    for r in $GUILE guile guile3 guile3.0 guile-3.0 guile2 guile2.2 guile2.0 guile-2.0 \
              guile1 guile19 guile18 \
              guile1.9 guile1.8 \
              guile-1 guile-1.9 guile-1.8; do
@@ -577,6 +577,9 @@ AC_DEFUN(STEPMAKE_GUILE_DEVEL, [
         PKG_CHECK_MODULES([GUILE], [$GUILE_FLAVOR],
                             [true], [GUILE_FLAVOR="missing"])
     else
+        PKG_CHECK_MODULES([GUILE], [guile-3.0 >= 3.0.0],
+                            [GUILE_FLAVOR="guile-3.0"], [
+            AC_MSG_RESULT([no])
         PKG_CHECK_MODULES([GUILE], [guile-1.8 >= 1.8.2],
                             [GUILE_FLAVOR="guile-1.8"], [
             PKG_CHECK_MODULES([GUILE], [guile-2.2 >= 2.2.0],
@@ -584,6 +587,7 @@ AC_DEFUN(STEPMAKE_GUILE_DEVEL, [
                 GUILE_FLAVOR="missing"
             ])
         ])
+        ])
     fi
 
     case "$GUILE_FLAVOR" in
