--- lilypond-2.8.6/Documentation/user/converters.itely.orig	2006-05-09 09:29:11.000000000 +0200
+++ lilypond-2.8.6/Documentation/user/converters.itely	2006-08-31 21:34:37.783341250 +0200
@@ -22,17 +22,17 @@
 
 
 @menu
-* Invoking midi2ly::            Importing MIDI.
-* Invoking etf2ly::             Importing Finale.
-* Invoking musicxml2ly::        
-* Invoking abc2ly::             Importing ABC.          
-* Invoking mup2ly::             Importing MUP. 
+* midi2ly::                     Importing MIDI.
+* etf2ly::                      Importing Finale.
+* musicxml2ly::                 
+* abc2ly::                      Importing ABC.
+* mup2ly::                      Importing MUP.
 * Generating LilyPond files::   
 @end menu
 
 
 
-@node Invoking midi2ly
+@node midi2ly
 @section Invoking @command{midi2ly}
 
 @cindex MIDI
@@ -119,7 +119,7 @@
 to a single duration and add phrase markings or pedal indicators.
 
 
-@node Invoking etf2ly
+@node etf2ly
 @section Invoking @command{etf2ly}
 
 @cindex ETF
@@ -158,7 +158,7 @@
 The list of articulation scripts is incomplete.  Empty measures
 confuse @command{etf2ly}.  Sequences of grace notes are ended improperly.
 
-@node Invoking musicxml2ly
+@node musicxml2ly
 @section Invoking @code{musicxml2ly}
 
 @uref{http://@/www.@/recordarde@/.com/xml@/.html,MusicXML} is an XML dialect
@@ -182,7 +182,7 @@
 print version information.
 @end table
 
-@node Invoking abc2ly
+@node abc2ly
 @section Invoking @code{abc2ly}
 
 @cindex ABC
@@ -243,7 +243,7 @@
 @command{abc2ly} ignores the ABC beaming.
 
 
-@node Invoking mup2ly
+@node mup2ly
 @section Invoking @command{mup2ly}
 
 Mup (Music Publisher) is a shareware music notation program by Arkkra
--- lilypond-2.8.6/Documentation/user/invoking.itely.orig	2006-05-09 09:29:11.000000000 +0200
+++ lilypond-2.8.6/Documentation/user/invoking.itely	2006-08-31 21:36:42.315124000 +0200
@@ -8,7 +8,7 @@
 
 
 @menu
-* Invoking lilypond::           
+* lilypond::           
 * Notes for the MacOS X app::   
 * Updating files with convert-ly::  
 * Reporting bugs::              
@@ -17,7 +17,7 @@
 * Point and click::             
 @end menu
 
-@node Invoking lilypond
+@node lilypond
 @section Invoking lilypond
 @cindex Invoking LilyPond
 @cindex command line options
--- lilypond-2.8.6/Documentation/user/lilypond-book.itely.orig	2006-05-02 13:25:52.000000000 +0200
+++ lilypond-2.8.6/Documentation/user/lilypond-book.itely	2006-08-31 21:37:00.248244750 +0200
@@ -36,7 +36,7 @@
 * Integrating Texinfo and music::
 * Integrating HTML and music::
 * Music fragment options::
-* Invoking lilypond-book::
+* lilypond-book::
 * Filename extensions::
 @end menu
 
@@ -294,7 +294,7 @@
 
 @noindent
 where the @var{file}@command{psfonts} file is obtained from
-@command{lilypond-book}, @xref{Invoking lilypond-book}, for details. PDF
+@command{lilypond-book}, @xref{lilypond-book}, for details. PDF
 can then be produced with a PostScript to PDF translator like
 @code{ps2pdf} (which is part of GhostScript).  Running @command{dvips}
 will produce some warnings about fonts; these are harmless and may
@@ -557,7 +557,7 @@
 @end table
 
 
-@node Invoking lilypond-book
+@node lilypond-book
 @section Invoking @command{lilypond-book}
 
 @command{lilypond-book} produces a file with one of the following
@@ -677,7 +677,7 @@
 
 You can use any filename extension for the input file, but if you do not
 use the recommended extension for a particular format you may need to
-manually specify the output format.  @xref{Invoking lilypond-book}, for
+manually specify the output format.  @xref{lilypond-book}, for
 details.  Otherwise, @command{lilypond-book} automatically selects the
 output format based on the input filename's extension.
 
--- lilypond-2.8.6/Documentation/user/lilypond.tely.orig	2006-05-09 09:29:11.000000000 +0200
+++ lilypond-2.8.6/Documentation/user/lilypond.tely	2006-08-31 21:38:18.621142750 +0200
@@ -5,15 +5,15 @@
 @afourpaper
 @end iftex
 
-@dircategory GNU music project
+@dircategory Miscellaneous:
 @direntry
-* LilyPond: (lilypond/lilypond).           The GNU music typesetter.
-* abc2ly: (lilypond/lilypond)Invoking abc2ly.          Importing ABC.          
-* convert-ly: (lilypond/lilypond)Invoking convert-ly.  Older LilyPond versions.
-* etf2ly: (lilypond/lilypond)Invoking etf2ly.          Importing Finale.
-* lilypond-book: (lilypond/lilypond)LilyPond-book.     Itegrating text and music.
-* midi2ly: (lilypond/lilypond)Invoking midi2ly.        Importing MIDI.
-* mup2ly: (lilypond/lilypond)Invoking mup2ly.          Importing Mup. 
+* LilyPond: (lilypond/lilypond).	The GNU music typesetter
+* abc2ly: (lilypond/lilypond)abc2ly.	Importing ABC
+* convert-ly: (lilypond/lilypond)convert-ly.	Older LilyPond versions
+* etf2ly: (lilypond/lilypond)etf2ly.	Importing Finale
+* lilypond-book: (lilypond/lilypond)LilyPond-book.	Itegrating text and music
+* midi2ly: (lilypond/lilypond)midi2ly.	Importing MIDI
+* mup2ly: (lilypond/lilypond)mup2ly.	Importing Mup
 @end direntry
 
 
--- lilypond-2.6.4/Documentation/user/music-glossary.tely.orig	2005-08-25 13:12:54.000000000 +0200
+++ lilypond-2.6.4/Documentation/user/music-glossary.tely	2005-10-29 20:36:59.482078520 +0200
@@ -6,9 +6,9 @@
 @documentencoding utf-8
 @documentlanguage en
 
-@dircategory GNU music project
+@dircategory Miscellaneous:
 @direntry
-* Glossary: (lilypond/music-glossary).  Glossary of music terms.
+* Glossary: (music-glossary).		Glossary of music terms
 @end direntry
 
 @ifhtml
--- lilypond-2.6.4/scm/documentation-lib.scm.orig	2005-06-14 02:44:14.000000000 +0200
+++ lilypond-2.6.4/scm/documentation-lib.scm	2005-10-29 20:44:44.211428920 +0200
@@ -111,10 +111,10 @@
    "\\input texinfo @c -*-texinfo-*-"
    "\n@setfilename " file-name ".info"
    "\n@settitle " name
-   "\n@dircategory GNU music project"
+   "\n@dircategory Miscellaneous:"
    "\n@direntry"
    ;; prepend GNU for dir, must be unique
-   "\n* GNU " name ": (" file-name ").          " name "."
+   "\n* GNU " name ": (" file-name ").	" name
    "\n@end direntry\n"
    "@documentlanguage en\n"
    "@documentencoding utf-8\n"))
