--- lilypond-2.6.4/python/GNUmakefile.orig	2005-06-16 13:54:02.000000000 +0200
+++ lilypond-2.6.4/python/GNUmakefile	2005-10-30 01:08:25.759188064 +0200
@@ -10,9 +10,9 @@
 INSTALLATION_OUT_SUFFIXES=1
 
 INSTALLATION_OUT_FILES=$(OUT_SO_MODULES)
-INSTALLATION_OUT_DIR=$(local_lilypond_datadir)/python
+INSTALLATION_OUT_DIR=$(local_lilypond_libdir)/python
 
-INSTALLATION_OUT_DIR1=$(local_lilypond_datadir)/python
+INSTALLATION_OUT_DIR1=$(local_lilypond_datadir)/python
 INSTALLATION_OUT_FILES1=$(OUT_PY_MODULES) $(OUT_PYC_MODULES)
 
 
