--- lilypond-2.6.4/aclocal.m4.orig	2005-10-09 21:54:56.000000000 +0200
+++ lilypond-2.6.4/aclocal.m4	2005-10-29 23:57:24.621979384 +0200
@@ -424,7 +424,7 @@
 EOF
 	FLEXLEXER_PATH=`$CXX -E conftest.cc | \
 	  sed 's!# 1 "\(.*\)FlexLexer.h"!@FLEXLEXER@\1@@!g' | grep '@@' | \
-	  sed 's!.*@FLEXLEXER@\(.*\)@@.*$!\1!g' ` >& /dev/null
+	  sed 's!.*@FLEXLEXER@\(.*\)@@.*$!\1!g' ` >/dev/null 2>&1
 	rm conftest.cc
 	AC_SUBST(FLEXLEXER_PATH)
         AC_MSG_RESULT($FLEXLEXER_PATH)
