--- lilypond-2.6.4/Documentation/user/converters.itely.orig	2005-09-23 23:20:58.000000000 +0200
+++ lilypond-2.6.4/Documentation/user/converters.itely	2005-10-29 20:32:46.600522360 +0200
@@ -12,16 +12,16 @@
 
 
 @menu
-* Invoking midi2ly::            Importing MIDI.
-* Invoking etf2ly::             Importing Finale.
-* Invoking abc2ly::             Importing ABC.          
-* Invoking mup2ly::             Importing MUP. 
+* midi2ly::                     Importing MIDI.
+* etf2ly::                      Importing Finale.
+* abc2ly::                      Importing ABC.          
+* mup2ly::                      Importing MUP. 
 * Generating LilyPond files::   
 @end menu
 
 
 
-@node Invoking midi2ly
+@node midi2ly
 @section Invoking @command{midi2ly}
 
 @cindex MIDI
@@ -105,7 +105,7 @@
 to a single duration and add phrase markings or pedal indicators.
 
 
-@node Invoking etf2ly
+@node etf2ly
 @section Invoking @command{etf2ly}
 
 @cindex ETF
@@ -141,7 +141,7 @@
 confuse @command{etf2ly}.  Sequences of grace notes are ended improperly.
 
 
-@node Invoking abc2ly
+@node abc2ly
 @section Invoking @code{abc2ly}
 
 @cindex ABC
@@ -202,7 +202,7 @@
 @command{abc2ly} ignores the ABC beaming.
 
 
-@node Invoking mup2ly
+@node mup2ly
 @section Invoking @command{mup2ly}
 
 Mup (Music Publisher) is a shareware music notation program by Arkkra
--- lilypond-2.6.4/Documentation/user/invoking.itely.orig	2005-09-23 23:20:58.000000000 +0200
+++ lilypond-2.6.4/Documentation/user/invoking.itely	2005-10-29 20:33:32.703513640 +0200
@@ -6,7 +6,7 @@
 
 
 @menu
-* Invoking lilypond::           
+* lilypond::           
 * Error messages::              
 * Updating files with convert-ly::  
 * Reporting bugs::              
@@ -15,7 +15,7 @@
 * Including LilyPond files::    
 @end menu
 
-@node Invoking lilypond
+@node lilypond
 @section Invoking lilypond
 @cindex Invoking LilyPond
 @cindex command line options
--- lilypond-2.6.4/Documentation/user/lilypond-book.itely.orig	2005-09-23 23:20:58.000000000 +0200
+++ lilypond-2.6.4/Documentation/user/lilypond-book.itely	2005-10-29 20:34:04.617661952 +0200
@@ -36,7 +36,7 @@
 * Integrating Texinfo and music::
 * Integrating HTML and music::
 * Music fragment options::
-* Invoking lilypond-book::
+* lilypond-book::
 * Filename extensions::
 @end menu
 
@@ -310,7 +310,7 @@
 
 @noindent
 where the @var{file}@command{psfonts} file is obtained from
-@command{lilypond-book}, @xref{Invoking lilypond-book} for details. PDF
+@command{lilypond-book}, @xref{lilypond-book} for details. PDF
 can then be produced with a PostScript to PDF translator like
 @code{ps2pdf} (which is part of GhostScript).  Running @command{dvips}
 will produce some warnings about fonts; these are harmless and may
@@ -573,7 +573,7 @@
 @end table
 
 
-@node Invoking lilypond-book
+@node lilypond-book
 @section Invoking @command{lilypond-book}
 
 @command{lilypond-book} produces a file with one of the following
@@ -689,7 +689,7 @@
 
 You can use any filename extension for the input file, but if you do not
 use the recommended extension for a particular format you may need to
-manually specify the output format.  @xref{Invoking lilypond-book}, for
+manually specify the output format.  @xref{lilypond-book}, for
 details.  Otherwise, @command{lilypond-book} automatically selects the
 output format based on the input filename's extension.
 
--- lilypond-2.6.4/Documentation/user/lilypond.tely.orig	2005-09-23 23:20:58.000000000 +0200
+++ lilypond-2.6.4/Documentation/user/lilypond.tely	2005-10-29 20:36:31.159384224 +0200
@@ -5,7 +5,7 @@
 @afourpaper
 @end iftex
 
-@dircategory GNU music project
+@dircategory Miscellaneous:
 @direntry
 
 @ignore
@@ -35,14 +35,14 @@
   
 @end ignore
 
-* LilyPond: (lilypond/lilypond).           The GNU music typesetter.
-* abc2ly: (lilypond/lilypond)Invoking abc2ly.          Importing ABC.          
-* convert-ly: (lilypond/lilypond)Invoking convert-ly.  Older LilyPond versions.
-* etf2ly: (lilypond/lilypond)Invoking etf2ly.          Importing Finale.
-* lilypond-book: (lilypond/lilypond)LilyPond-book.     Itegrating text and music.
-* midi2ly: (lilypond/lilypond)Invoking midi2ly.        Importing MIDI.
-* mup2ly: (lilypond/lilypond)Invoking mup2ly.          Importing Mup. 
-* musedata2ly: (lilypond/lilypond)Invoking musedata2ly.        Importing MuseData.
+* LilyPond: (lilypond).			The GNU music typesetter
+* abc2ly: (lilypond)abc2ly.		Importing ABC
+* convert-ly: (lilypond)convert-ly.	Older LilyPond versions
+* etf2ly: (lilypond)etf2ly.		Importing Finale
+* lilypond-book: (lilypond)LilyPond-book.	Itegrating text and music
+* midi2ly: (lilypond)midi2ly.		Importing MIDI
+* mup2ly: (lilypond)mup2ly.		Importing Mup
+* musedata2ly: (lilypond)musedata2ly.	Importing MuseData
 @end direntry
 
 
--- lilypond-2.6.4/Documentation/user/music-glossary.tely.orig	2005-08-25 13:12:54.000000000 +0200
+++ lilypond-2.6.4/Documentation/user/music-glossary.tely	2005-10-29 20:36:59.482078520 +0200
@@ -6,9 +6,9 @@
 @documentencoding utf-8
 @documentlanguage en
 
-@dircategory GNU music project
+@dircategory Miscellaneous:
 @direntry
-* Glossary: (lilypond/music-glossary).  Glossary of music terms.
+* Glossary: (music-glossary).		Glossary of music terms
 @end direntry
 
 @ifhtml
--- lilypond-2.6.4/scm/documentation-lib.scm.orig	2005-06-14 02:44:14.000000000 +0200
+++ lilypond-2.6.4/scm/documentation-lib.scm	2005-10-29 20:44:44.211428920 +0200
@@ -111,10 +111,10 @@
    "\\input texinfo @c -*-texinfo-*-"
    "\n@setfilename " file-name ".info"
    "\n@settitle " name
-   "\n@dircategory GNU music project"
+   "\n@dircategory Miscellaneous:"
    "\n@direntry"
    ;; prepend GNU for dir, must be unique
-   "\n* GNU " name ": (" file-name ").          " name "."
+   "\n* GNU " name ": (" file-name ").	" name
    "\n@end direntry\n"
    "@documentlanguage en\n"
    "@documentencoding utf-8\n"))
