diff -up lilypond-2.10.33/flower/file-name.cc.gcc43 lilypond-2.10.33/flower/file-name.cc
--- lilypond-2.10.33/flower/file-name.cc.gcc43	2008-04-08 00:13:27.000000000 -0400
+++ lilypond-2.10.33/flower/file-name.cc	2008-04-08 00:13:55.000000000 -0400
@@ -10,8 +10,10 @@
 #include "file-name.hh"
 
 #include <cstdio>
+#include <cstdlib>
 #include <cerrno>
 #include <unistd.h>
+#include <limits.h>
 
 using namespace std;
 
diff -up lilypond-2.10.33/flower/include/rational.hh.gcc43 lilypond-2.10.33/flower/include/rational.hh
--- lilypond-2.10.33/flower/include/rational.hh.gcc43	2008-04-08 00:14:23.000000000 -0400
+++ lilypond-2.10.33/flower/include/rational.hh	2008-04-08 00:15:09.000000000 -0400
@@ -9,6 +9,8 @@
 #ifndef RATIONAL_HH
 #define RATIONAL_HH
 
+#include <climits>
+
 #include "std-string.hh"
 
 #include "std-string.hh"
diff -up lilypond-2.10.33/flower/include/std-vector.hh.gcc43 lilypond-2.10.33/flower/include/std-vector.hh
--- lilypond-2.10.33/flower/include/std-vector.hh.gcc43	2008-04-08 00:11:55.000000000 -0400
+++ lilypond-2.10.33/flower/include/std-vector.hh	2008-04-08 00:12:58.000000000 -0400
@@ -22,6 +22,7 @@
 #include <algorithm>   /* find, reverse, sort */
 #include <functional>  /* unary_function */
 #include <cassert>
+#include <string>
 
 using namespace std;
 
diff -up lilypond-2.10.33/lily/midi-stream.cc.gcc43 lilypond-2.10.33/lily/midi-stream.cc
--- lilypond-2.10.33/lily/midi-stream.cc.gcc43	2008-04-08 00:19:35.000000000 -0400
+++ lilypond-2.10.33/lily/midi-stream.cc	2008-04-08 00:19:44.000000000 -0400
@@ -9,6 +9,7 @@
 #include "midi-stream.hh"
 
 #include <cerrno>
+#include <cstring>
 using namespace std;
 
 #include "international.hh"
diff -up lilypond-2.10.33/lily/piano-pedal-engraver.cc.gcc43 lilypond-2.10.33/lily/piano-pedal-engraver.cc
--- lilypond-2.10.33/lily/piano-pedal-engraver.cc.gcc43	2008-04-08 00:21:48.000000000 -0400
+++ lilypond-2.10.33/lily/piano-pedal-engraver.cc	2008-04-08 00:22:09.000000000 -0400
@@ -10,6 +10,8 @@
   bracketed pedals.
 */
 
+#include <cstring>
+
 #include "engraver.hh"
 
 #include "axis-group-interface.hh"
diff -up lilypond-2.10.33/lily/prob.cc.gcc43 lilypond-2.10.33/lily/prob.cc
--- lilypond-2.10.33/lily/prob.cc.gcc43	2008-04-08 00:22:39.000000000 -0400
+++ lilypond-2.10.33/lily/prob.cc	2008-04-08 00:22:50.000000000 -0400
@@ -6,6 +6,8 @@
   (c) 2004--2007 Jan Nieuwenhuizen <janneke@gnu.org>
 */
 
+#include <cstring>
+
 #include "prob.hh"
 #include "main.hh"
 #include "item.hh"
diff -up lilypond-2.10.33/lily/relocate.cc.gcc43 lilypond-2.10.33/lily/relocate.cc
--- lilypond-2.10.33/lily/relocate.cc.gcc43	2008-04-08 00:23:10.000000000 -0400
+++ lilypond-2.10.33/lily/relocate.cc	2008-04-08 00:23:23.000000000 -0400
@@ -16,6 +16,7 @@
 
 #include <sys/types.h>
 #include <dirent.h>
+#include <cstring>
 
 #if HAVE_GETTEXT
 #include <libintl.h>
diff -up lilypond-2.10.33/lily/source-file.cc.gcc43 lilypond-2.10.33/lily/source-file.cc
--- lilypond-2.10.33/lily/source-file.cc.gcc43	2008-04-08 00:24:09.000000000 -0400
+++ lilypond-2.10.33/lily/source-file.cc	2008-04-08 00:24:48.000000000 -0400
@@ -18,6 +18,7 @@
 #endif
 
 #include <cstdio>
+#include <cstring>
 
 #if HAVE_SSTREAM
 #include <sstream>
diff -up lilypond-2.10.33/lily/translator.cc.gcc43 lilypond-2.10.33/lily/translator.cc
--- lilypond-2.10.33/lily/translator.cc.gcc43	2008-04-08 00:25:53.000000000 -0400
+++ lilypond-2.10.33/lily/translator.cc	2008-04-08 00:26:05.000000000 -0400
@@ -6,6 +6,8 @@
   (c) 1997--2007 Han-Wen Nienhuys <hanwen@xs4all.nl>
 */
 
+#include <cstring>
+
 #include "translator.hh"
 
 #include "context-def.hh"
