--- lilypond-2.8.6/aclocal.m4.orig	2006-08-31 21:59:48.001724000 +0200
+++ lilypond-2.8.6/aclocal.m4	2006-08-31 21:59:39.461190250 +0200
@@ -111,7 +111,7 @@
     num=`STEPMAKE_NUMERIC_VERSION($ver)`
     req=`STEPMAKE_NUMERIC_VERSION($3)`
     AC_MSG_RESULT([$ver])
-    if test "$num" -lt "$req"; then
+    if /usr/bin/test "$num" -lt "$req"; then
 	STEPMAKE_ADD_ENTRY($2, ["$r >= $3 (installed: $ver)"])
     fi
     vervar="`echo $1 | tr '[a-z]' '[A-Z]'`_VERSION"
