--- lilypond-2.20.0/aclocal.m4.orig	2020-06-21 17:28:04.690857844 +0200
+++ lilypond-2.20.0/aclocal.m4	2020-06-21 18:15:06.675569841 +0200
@@ -110,7 +110,7 @@
     num=`STEPMAKE_NUMERIC_VERSION($ver)`
     min=`STEPMAKE_NUMERIC_VERSION($3)`
     AC_MSG_RESULT([$ver])
-    if test "$num" -lt "$min"; then
+    if /usr/bin/test "$num" -lt "$min"; then
         STEPMAKE_ADD_ENTRY($2, ["$r >= $3 (installed: $ver)"])
     fi
     if test -n "$4"; then
