--- lilypond-2.4.0/scripts/lilypond-book.py	2004-10-29 20:30:26.000000000 +0000
+++ lilypond-2.4.0.new/scripts/lilypond-book.py	2004-11-04 08:42:04.329420488 +0000
@@ -717,7 +717,7 @@
 	cmd = r'latex "\nonstopmode \input %s"' % file
 	# Ugh.  (La)TeX writes progress and error messages on stdout
 	# Redirect to stderr
-	cmd = '(( %s >&2 ) >&- )' % cmd
+	cmd = '( %s  >&2 )' % cmd
 	ly.system (cmd)
 	ly.system ('dvips -Ppdf -u+ec-mftrace.map -u+lilypond.map -E -o %s.eps %s' \
 		   % (file, file))
--- lilypond-2.4.0/scripts/lilypond-latex.py	2004-10-10 11:42:06.000000000 +0000
+++ lilypond-2.4.0.new/scripts/lilypond-latex.py	2004-11-04 08:42:26.056117528 +0000
@@ -446,7 +446,7 @@
 	
 	# Ugh.  (La)TeX writes progress and error messages on stdout
 	# Redirect to stderr
-	cmd = '(( %s  >&2 ) >&- )' % cmd
+	cmd = '( %s  >&2 )' % cmd
 	status = ly.system (cmd, ignore_error = 1)
 	signal = 0xf & status
  	exit_status = status >> 8
