--- lilypond-1.6.11/stepmake/aclocal.m4.orig	2003-06-23 00:39:22.000000000 +0200
+++ lilypond-1.6.11/stepmake/aclocal.m4	2003-08-24 16:18:27.620831152 +0200
@@ -855,7 +855,7 @@
 
 
 AC_DEFUN(STEPMAKE_PYTHON_DEVEL, [
-    AC_HAVE_HEADERS(python2.2/Python.h python2.1/Python.h python2.0/Python.h python2/Python.h python/Python.h python1.5/Python.h Python.h, PYTHON_HEADER=yes)
+    AC_HAVE_HEADERS(python2.3/Python.h python2.2/Python.h python2.1/Python.h python2.0/Python.h python2/Python.h python/Python.h python1.5/Python.h Python.h, PYTHON_HEADER=yes)
     if test -z "$PYTHON_HEADER"; then
 	warn='python.h (python-devel, python-dev or libpython-dev package)'
 	STEPMAKE_ADD_ENTRY($1, $warn)
--- lilypond-1.6.11/python/midi.c.orig	2002-02-10 18:43:09.000000000 +0100
+++ lilypond-1.6.11/python/midi.c	2003-08-24 16:59:03.040591072 +0200
@@ -23,6 +23,8 @@
 /* urg */
 #if HAVE_PYTHON2_PYTHON_H
 #include <python2/Python.h>
+#elif HAVE_PYTHON2_3_PYTHON_H
+#include <python2.3/Python.h>
 #elif HAVE_PYTHON2_2_PYTHON_H
 #include <python2.2/Python.h>
 #elif HAVE_PYTHON2_1_PYTHON_H
--- lilypond-1.6.11/config.hh.in.orig	2003-05-16 01:06:57.000000000 +0200
+++ lilypond-1.6.11/config.hh.in	2003-08-24 17:07:57.235381120 +0200
@@ -46,6 +46,9 @@
 /* define if you have kpse_find_tfm */
 #define HAVE_KPSE_FIND_TFM 0
 
+/* define if you have python2.3/Python.h header */
+#define HAVE_PYTHON2_3_PYTHON_H 0
+
 /* define if you have python2.2/Python.h header */
 #define HAVE_PYTHON2_2_PYTHON_H 0
 
