--- lilypond-2.2.0/Documentation/user/converters.itely.orig	2004-03-10 01:53:02.000000000 +0100
+++ lilypond-2.2.0/Documentation/user/converters.itely	2004-04-26 00:03:52.888374112 +0200
@@ -12,16 +12,16 @@
 
 
 @menu
-* Invoking convert-ly::         Older LilyPond versions.
-* Invoking midi2ly::            Importing MIDI.
-* Invoking etf2ly::             Importing Finale.
-* Invoking abc2ly::             Importing ABC.          
-* Invoking musedata2ly::        Importing Musedata.
-* Invoking mup2ly::             Importing MUP. 
+* convert-ly::                  Older LilyPond versions.
+* midi2ly::                     Importing MIDI.
+* etf2ly::                      Importing Finale.
+* abc2ly::                      Importing ABC.          
+* musedata2ly::                 Importing Musedata.
+* mup2ly::                      Importing MUP. 
 @end menu
 
 
-@node Invoking convert-ly
+@node convert-ly
 @section Invoking convert-ly
 
 Convert-ly sequentially applies different conversions to upgrade a
@@ -66,7 +66,7 @@
 
 Not all language changes are handled. Only one output option can be specified.
 
-@node Invoking midi2ly
+@node midi2ly
 @section Invoking midi2ly
 
 @cindex MIDI
@@ -131,7 +131,7 @@
 @end table
 
 
-@node Invoking etf2ly
+@node etf2ly
 @section Invoking etf2ly
 
 @cindex ETF
@@ -165,7 +165,7 @@
 etf2ly. Sequences of grace notes are ended improperly sometimes. 
 
 
-@node Invoking abc2ly
+@node abc2ly
 @section Invoking abc2ly
 
 @cindex ABC
@@ -223,7 +223,7 @@
 
 abc2ly ignores the ABC beaming.
 
-@node Invoking musedata2ly
+@node musedata2ly
 @section Invoking musedata2ly
 
 @cindex MuseData
@@ -258,7 +258,7 @@
 
 @file{musedata2ly} converts only a small subset of MuseData. 
 
-@node Invoking mup2ly
+@node mup2ly
 @section Invoking mup2ly
 
 Mup (Music Publisher) is a shareware music notation program by Arkkra
--- lilypond-2.2.0/Documentation/user/introduction.itely.orig	2004-03-31 10:41:13.000000000 +0200
+++ lilypond-2.2.0/Documentation/user/introduction.itely	2004-04-26 00:04:28.757921112 +0200
@@ -662,7 +662,7 @@
 @ifhtml
 The chapter
 @end ifhtml
-@emph{@ref{Invoking LilyPond}} shows how to run LilyPond and its helper
+@emph{@ref{LilyPond}} shows how to run LilyPond and its helper
 programs.
 
 @item
--- lilypond-2.2.0/Documentation/user/invoking.itexi.orig	2004-03-22 15:52:21.000000000 +0100
+++ lilypond-2.2.0/Documentation/user/invoking.itexi	2004-04-26 00:05:19.864151784 +0200
@@ -1,12 +1,12 @@
 @c -*-texinfo-*-
-@node Invoking LilyPond
+@node LilyPond
 @chapter Invoking LilyPond
 
 This chapter details the technicalities of running LilyPond.
 
  
 @menu
-* Invoking lilypond::           Titling LilyPond scores.
+* lilypond::                    Titling LilyPond scores.
 * Invoking the lilypond binary::  
 * Error messages::              
 * Reporting bugs::              
@@ -14,7 +14,7 @@
 * Point and click::             
 @end menu
 
-@node Invoking lilypond
+@node lilypond
 @section Invoking lilypond
 
 Nicely titled output is created through a separate program:
--- lilypond-2.2.0/Documentation/user/lilypond-book.itely.orig	2004-03-31 14:24:04.000000000 +0200
+++ lilypond-2.2.0/Documentation/user/lilypond-book.itely	2004-04-26 00:05:43.964487976 +0200
@@ -40,7 +40,7 @@
 * Integrating LaTeX and music::  
 * Integrating HTML and music::  
 * Music fragment options::      
-* Invoking lilypond-book::      
+* lilypond-book::      
 @end menu
 
 
@@ -247,7 +247,7 @@
   <lilypondfile>trip.ly</lilypondfile>
 @end example
 This runs @file{trip.ly} through @code{lilypond} (see also
-@ref{Invoking lilypond}), and substitutes a preview image in the
+@ref{lilypond}), and substitutes a preview image in the
 output. The image links to a separate HTML file, so clicking it will
 take the viewer to a menu, with links to images, midi and printouts.
 
@@ -340,7 +340,7 @@
 @end table
 
 
-@node Invoking lilypond-book
+@node lilypond-book
 @section Invoking lilypond-book
 
 
--- lilypond-2.2.0/Documentation/user/lilypond.tely.orig	2004-03-30 01:24:42.000000000 +0200
+++ lilypond-2.2.0/Documentation/user/lilypond.tely	2004-04-26 00:07:51.915036536 +0200
@@ -12,19 +12,19 @@
     
 @end ignore
 
-@dircategory GNU music project
+@dircategory Miscellaneous:
 @direntry
 @c  prepend GNU for dir, must be unique
 @c  do not list the `lilypond' node at toplevel, so that `info lilypond'
 @c  goes to Top.
 @c  * lilypond: (lilypond/lilypond)Invoking LilyPond.      Titling LilyPond scores.
-* GNU LilyPond: (lilypond/lilypond).           The GNU music typesetter.
-* convert-ly: (lilypond/lilypond)Invoking convert-ly.  Older LilyPond versions.
-* midi2ly: (lilypond/lilypond)Invoking midi2ly.        Importing MIDI.
-* etf2ly: (lilypond/lilypond)Invoking etf2ly.          Importing Finale.
-* abc2ly: (lilypond/lilypond)Invoking abc2ly.          Importing ABC.          
-* musedata2ly: (lilypond/lilypond)Invoking musedata2ly.        Importing MuseData.
-* mup2ly: (lilypond/lilypond)Invoking mup2ly.          Importing Mup. 
+* GNU LilyPond: (lilypond/lilypond).	The GNU music typesetter
+* convert-ly: (lilypond/lilypond)convert-ly.  Older LilyPond versions
+* midi2ly: (lilypond/lilypond)midi2ly.	Importing MIDI
+* etf2ly: (lilypond/lilypond)etf2ly.	Importing Finale
+* abc2ly: (lilypond/lilypond)abc2ly.	Importing ABC
+* musedata2ly: (lilypond/lilypond)musedata2ly.  Importing MuseData
+* mup2ly: (lilypond/lilypond)mup2ly.	Importing Mup
 @end direntry
 
 
@@ -162,7 +162,7 @@
 * Notation manual::                All notation supported, and how to
                                    produce it.
 * Changing defaults::              Tuning output.
-* Invoking LilyPond::              Operation.
+* LilyPond::                       Operation.
 * Converting from other formats::  Converting to lilypond source format.
 * lilypond-book manual::           Integrating text and music with lilypond-book.
 * Unified index::                  
--- lilypond-2.2.0/Documentation/user/tutorial.itely.orig	2004-04-01 12:22:10.000000000 +0200
+++ lilypond-2.2.0/Documentation/user/tutorial.itely	2004-04-26 00:08:28.000550704 +0200
@@ -985,7 +985,7 @@
 
 These annotations make future upgrades of LilyPond go more
 smoothly. Changes in the syntax are handled with a special program,
-@file{convert-ly} (see @ref{Invoking convert-ly}), and it uses
+@file{convert-ly} (see @ref{convert-ly}), and it uses
 @code{\version} to determine what rules to apply.
 
 
@@ -1266,7 +1266,7 @@
 @}
 @end example
 
-More information on titling can be found in @ref{Invoking lilypond}.
+More information on titling can be found in @ref{lilypond}.
 
 
 @node Single staff polyphony
--- lilypond-2.2.0/Documentation/user/advanced.itely.orig	2004-03-08 01:36:46.000000000 +0100
+++ lilypond-2.2.0/Documentation/user/advanced.itely	2004-04-26 00:09:04.037072320 +0200
@@ -71,7 +71,7 @@
 
 This chapter discusses Steps@tie{}1. to@tie{}4. Step@tie{}5 is
 discussed in @ref{Tuning output}, while Step@tie{}6 is discussed in
-@ref{Invoking LilyPond}.
+@ref{LilyPond}.
 
 The diagram above shows how music typesetting has been split-up in
 multiple subproblems. However, splitting up problems always produces
--- lilypond-2.2.0/Documentation/user/changing-defaults.itely.orig	2004-03-31 11:04:14.000000000 +0200
+++ lilypond-2.2.0/Documentation/user/changing-defaults.itely	2004-04-26 00:09:15.876272488 +0200
@@ -2010,7 +2010,7 @@
 
 @seealso
 
-In this manual: @ref{Invoking lilypond}.
+In this manual: @ref{lilypond}.
 
 Examples: @inputfileref{input/regression,between-systems.ly}.
 
--- lilypond-2.2.0/scm/documentation-lib.scm.orig	2004-03-18 11:19:26.000000000 +0100
+++ lilypond-2.2.0/scm/documentation-lib.scm	2004-04-26 00:10:40.321434872 +0200
@@ -139,10 +139,10 @@
    "\\input texinfo @c -*-texinfo-*-"
    "\n@setfilename " file-name ".info"
    "\n@settitle " name
-   "\n@dircategory GNU music project"
+   "\n@dircategory Miscellaneous:"
    "\n@direntry"
    ;; prepend GNU for dir, must be unique
-   "\n* GNU " name ": (" file-name ").          " name "."
+   "\n* GNU " name ": (" file-name ").  " name
    "\n@end direntry"
    ))
 
--- lilypond-2.2.0/Documentation/user/music-glossary.tely.orig	2004-03-30 01:24:42.000000000 +0200
+++ lilypond-2.2.0/Documentation/user/music-glossary.tely	2004-04-26 00:44:21.894109112 +0200
@@ -2,9 +2,9 @@
 @settitle Music Glossary
 @setfilename music-glosssary.info
 
-@dircategory GNU music project
+@dircategory Miscellaneous:
 @direntry
-* Glossary: (lilypond/music-glossary).                  Glossary of music terms.
+* Glossary: (lilypond/music-glossary).	Glossary of music terms
 @end direntry
 
 @ifhtml
