--- lilypond-2.10.0/Documentation/user/converters.itely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/converters.itely	2006-11-20 20:07:34.356722750 +0100
@@ -25,16 +25,16 @@
 
 
 @menu
-* Invoking midi2ly::            Importing MIDI.
-* Invoking etf2ly::             Importing Finale.
-* Invoking musicxml2ly::        
-* Invoking abc2ly::             Importing ABC.      
+* midi2ly::                     Importing MIDI.
+* etf2ly::                      Importing Finale.
+* musicxml2ly::                 
+* abc2ly::                      Importing ABC.      
 * Generating LilyPond files::   
 @end menu
 
 
 
-@node Invoking midi2ly
+@node midi2ly
 @section Invoking @command{midi2ly}
 
 @cindex MIDI
@@ -121,7 +121,7 @@
 to a single duration and add phrase markings or pedal indicators.
 
 
-@node Invoking etf2ly
+@node etf2ly
 @section Invoking @command{etf2ly}
 
 @cindex ETF
@@ -160,7 +160,7 @@
 The list of articulation scripts is incomplete.  Empty measures
 confuse @command{etf2ly}.  Sequences of grace notes are ended improperly.
 
-@node Invoking musicxml2ly
+@node musicxml2ly
 @section Invoking @code{musicxml2ly}
 
 @uref{http://@/www.@/recordarde@/.com/xml@/.html,MusicXML} is an XML dialect
@@ -184,7 +184,7 @@
 print version information.
 @end table
 
-@node Invoking abc2ly
+@node abc2ly
 @section Invoking @code{abc2ly}
 
 @cindex ABC
--- lilypond-2.10.0/Documentation/user/running.itely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/running.itely	2006-11-20 20:08:46.477230000 +0100
@@ -20,7 +20,7 @@
 you are unfamiliar with the command-line.
 
 @menu
-* Invoking lilypond::           
+* lilypond::           
 * Notes for the MacOS X app::   
 * Updating files with convert-ly::  
 * Reporting bugs::              
@@ -29,7 +29,7 @@
 * Point and click::             
 @end menu
 
-@node Invoking lilypond
+@node lilypond
 @section Invoking lilypond
 @cindex Invoking LilyPond
 @cindex command line options
--- lilypond-2.10.0/Documentation/user/lilypond-book.itely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/lilypond-book.itely	2006-11-20 20:09:06.130458250 +0100
@@ -37,7 +37,7 @@
 * Integrating HTML and music::  
 * Integrating DocBook and music::
 * Music fragment options::      
-* Invoking lilypond-book::      
+* lilypond-book::      
 * Filename extensions::         
 * Many quotes of a large score::  
 * Inserting LilyPond output into other programs::  
@@ -299,7 +299,7 @@
 
 @noindent
 where the @var{file}@command{psfonts} file is obtained from
-@command{lilypond-book}, @xref{Invoking lilypond-book}, for details. PDF
+@command{lilypond-book}, @xref{lilypond-book}, for details. PDF
 can then be produced with a PostScript to PDF translator like
 @code{ps2pdf} (which is part of GhostScript).  Running @command{dvips}
 will produce some warnings about fonts; these are harmless and may
@@ -677,7 +677,7 @@
 @end table
 
 
-@node Invoking lilypond-book
+@node lilypond-book
 @section Invoking @command{lilypond-book}
 
 @command{lilypond-book} produces a file with one of the following
@@ -802,7 +802,7 @@
 
 You can use any filename extension for the input file, but if you do not
 use the recommended extension for a particular format you may need to
-manually specify the output format.  @xref{Invoking lilypond-book}, for
+manually specify the output format.  @xref{lilypond-book}, for
 details.  Otherwise, @command{lilypond-book} automatically selects the
 output format based on the input filename's extension.
 
--- lilypond-2.10.0/Documentation/user/lilypond.tely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/lilypond.tely	2006-11-20 20:15:39.171021750 +0100
@@ -26,15 +26,15 @@
 @c
 @c Info files are installed in subdirectories to allow images to be present. 
 @c 
-@dircategory LilyPond
+@dircategory LilyPond:
 @direntry
-* LilyPond: (lilypond/lilypond).           The GNU music typesetter.
-* abc2ly: (lilypond/lilypond)Invoking abc2ly.          Importing ABC.      
-* convert-ly: (lilypond/lilypond)Invoking convert-ly.  Older LilyPond versions.
-* etf2ly: (lilypond/lilypond)Invoking etf2ly.          Importing Finale.
-* lilypond-book: (lilypond/lilypond)LilyPond-book.     Itegrating text and music.
-* midi2ly: (lilypond/lilypond)Invoking midi2ly.        Importing MIDI.
-* mup2ly: (lilypond/lilypond)Invoking mup2ly.          Importing Mup.
+* LilyPond: (lilypond/lilypond).	The GNU music typesetter
+* abc2ly: (lilypond/lilypond)abc2ly.	Importing ABC
+* convert-ly: (lilypond/lilypond)convert-ly.	Older LilyPond versions
+* etf2ly: (lilypond/lilypond)etf2ly.	Importing Finale
+* lilypond-book: (lilypond/lilypond)LilyPond-book.	Itegrating text and music
+* midi2ly: (lilypond/lilypond)midi2ly.	Importing MIDI
+* mup2ly: (lilypond/lilypond)mup2ly.	Importing Mup
 @end direntry
 
 @c don't remove this comment.
--- lilypond-2.10.0/Documentation/user/music-glossary.tely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/music-glossary.tely	2006-11-20 20:16:07.108767750 +0100
@@ -6,9 +6,9 @@
 @documentlanguage en
 
 @c see lilypond.tely for info installation note
-@dircategory LilyPond
+@dircategory LilyPond:
 @direntry
-* Glossary: (lilypond/music-glossary).  Glossary of music terms.
+* Glossary: (lilypond/music-glossary).	Glossary of music terms
 @end direntry
 
 @ifhtml
--- lilypond-2.10.0/Documentation/user/non-music.itely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/non-music.itely	2006-11-20 22:02:52.933106750 +0100
@@ -178,7 +178,7 @@
 @code{-dclip-systems}. The clips are output as EPS files, and are
 converted to PDF and PNG if these formats are switched on as well.
 
-For more information on output formats, see @ref{Invoking lilypond}.
+For more information on output formats, see @ref{lilypond}.
 
 @seealso
 
--- lilypond-2.10.0/scm/documentation-lib.scm.orig	2006-11-10 14:44:26.000000000 +0100
+++ lilypond-2.10.0/scm/documentation-lib.scm	2006-11-20 20:16:58.303967250 +0100
@@ -111,10 +111,10 @@
    "\\input texinfo @c -*-texinfo-*-"
    "\n@setfilename " file-name ".info"
    "\n@settitle " name
-   "\n@dircategory LilyPond"
+   "\n@dircategory LilyPond:"
    "\n@direntry"
    ;; prepend GNU for dir, must be unique
-   "\n* GNU " name ": (" file-name ").          " name "."
+   "\n* GNU " name ": (" file-name ").	" name
    "\n@end direntry\n"
    "@documentlanguage en\n"
    "@documentencoding utf-8\n"))
