--- lilypond-2.10.0/Documentation/user/converters.itely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/converters.itely	2006-11-20 20:07:34.356722750 +0100
@@ -31,16 +31,16 @@
 bug reports will almost certainly not be resolved.
 
 @menu
-* Invoking midi2ly::            Importing MIDI.
-* Invoking musicxml2ly::        Importing MusicXML.
-* Invoking abc2ly::             Importing ABC.
-* Invoking etf2ly::             Importing Finale.
+* midi2ly::                     Importing MIDI.
+* musicxml2ly::                 Importing MusicXML.
+* abc2ly::                      Importing ABC.
+* etf2ly::                      Importing Finale.
 * Generating LilyPond files::   GUIs, transcribers, and algorithmic composition programs.
 @end menu
 
 
 
-@node Invoking midi2ly
+@node midi2ly
 @section Invoking @command{midi2ly}
 
 @cindex MIDI
@@ -127,7 +127,7 @@
 to a single duration and add phrase markings or pedal indicators.
 
 
-@node Invoking musicxml2ly
+@node musicxml2ly
 @section Invoking @code{musicxml2ly}
 
 @cindex MusicXML
@@ -195,7 +195,7 @@
 @end table
 
 
-@node Invoking abc2ly
+@node abc2ly
 @section Invoking @code{abc2ly}
 
 @cindex ABC
@@ -260,7 +260,7 @@
 @command{abc2ly} ignores the ABC beaming.
 
 
-@node Invoking etf2ly
+@node etf2ly
 @section Invoking @command{etf2ly}
 
 @cindex ETF
--- lilypond-2.10.0/Documentation/user/running.itely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/running.itely	2006-11-20 20:08:46.477230000 +0100
@@ -50,12 +50,12 @@
 if you are unfamiliar with the command-line.
 
 @menu
-* Invoking lilypond::
+* lilypond::
 * Command line options for lilypond::
 * Environment variables::
 @end menu
 
-@node Invoking lilypond
+@node lilypond
 @subsection Invoking @command{lilypond}
 
 @cindex Invoking @command{lilypond}
--- lilypond-2.10.0/Documentation/user/lilypond-book.itely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/lilypond-book.itely	2006-11-20 20:09:06.130458250 +0100
@@ -48,7 +48,7 @@
 * An example of a musicological document::
 * Integrating music and text::
 * Music fragment options::
-* Invoking lilypond-book::
+* lilypond-book::
 * Filename extensions::
 * Alternate methods of mixing text and music::
 @end menu
@@ -363,7 +363,7 @@
 @seealso
 There are specific @command{lilypond-book} command line options and
 other details to know when processing @LaTeX{} documents, see
-@ref{Invoking lilypond-book}.
+@ref{lilypond-book}.
 
 
 @node Texinfo
@@ -765,7 +765,7 @@
 @end table
 
 
-@node Invoking lilypond-book
+@node lilypond-book
 @section Invoking @command{lilypond-book}
 
 @command{lilypond-book} produces a file with one of the following
@@ -953,8 +953,8 @@
 
 You can use any filename extension for the input file, but if you do not
 use the recommended extension for a particular format you may need to
-manually specify the output format; for details, see @ref{Invoking
-lilypond-book}.  Otherwise, @command{lilypond-book} automatically
+manually specify the output format; for details, see @ref{lilypond-book}.
+Otherwise, @command{lilypond-book} automatically
 selects the output format based on the input filename's extension.
 
 @quotation
--- lilypond-2.10.0/Documentation/user/lilypond.tely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/lilypond.tely	2006-11-20 20:15:39.171021750 +0100
@@ -35,19 +35,19 @@
 @c
 @c Info files are no longer installed in a subdirectory, images are
 @c expected to be found in lilypond/ subdirectory.
-@dircategory GNU LilyPond --- the music typesetter
+@dircategory GNU LilyPond --- the music typesetter:
 @direntry
-* LilyPond Learning Manual: (lilypond-learning).  Start here.
-* Music Glossary: (music-glossary).               For non-English users.
-* LilyPond: (lilypond).                           LilyPond Notation Reference.
-* LilyPond Internals Reference: (lilypond-internals).  Definitions for tweaking.
-* LilyPond Application Usage: (lilypond-program). Installing and running applications.
-* abc2ly: (lilypond-program)Invoking abc2ly.      Importing ABC.      
-* convert-ly: (lilypond-program)Updating files with convert-ly. Older LilyPond versions.
-* etf2ly: (lilypond-program)Invoking etf2ly.      Importing Finale.
-* lilypond-book: (lilypond-program)LilyPond-book. Integrating text and music.
-* midi2ly: (lilypond-program)Invoking midi2ly.    Importing MIDI.
-* musicxml2ly: (lilypond-program)Invoking musicxml2ly.  Importing MusicXML.
+* LilyPond Learning Manual: (lilypond-learning).  Start here
+* Music Glossary: (music-glossary).               For non-English users
+* LilyPond: (lilypond).                           LilyPond Notation Reference
+* LilyPond Internals Reference: (lilypond-internals).  Definitions for tweaking
+* LilyPond Application Usage: (lilypond-program). Installing and running applications
+* abc2ly: (lilypond-program)abc2ly.               Importing ABC
+* convert-ly: (lilypond-program)Updating files with convert-ly. Older LilyPond versions
+* etf2ly: (lilypond-program)etf2ly.               Importing Finale
+* lilypond-book: (lilypond-program)LilyPond-book. Integrating text and music
+* midi2ly: (lilypond-program)midi2ly.             Importing MIDI
+* musicxml2ly: (lilypond-program)musicxml2ly.     Importing MusicXML
 @end direntry
 
 @c don't remove this comment.
--- lilypond-2.10.0/Documentation/user/music-glossary.tely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/music-glossary.tely	2006-11-20 20:16:07.108767750 +0100
@@ -6,9 +6,9 @@
 @documentlanguage en
 
 @c see lilypond.tely for info installation note
-@dircategory LilyPond
+@dircategory LilyPond:
 @direntry
-* Glossary: (lilypond/music-glossary).  Glossary of music terms.
+* Glossary: (lilypond/music-glossary).	Glossary of music terms
 @end direntry
 
 @ifhtml
--- lilypond-2.10.0/Documentation/user/input.itely.orig	2006-11-10 14:44:25.000000000 +0100
+++ lilypond-2.10.0/Documentation/user/input.itely	2006-11-20 22:02:52.933106750 +0100
@@ -1324,7 +1324,7 @@
 @code{-dclip-systems}.  The clips are output as EPS files, and are
 converted to PDF and PNG if these formats are switched on as well.
 
-For more information on output formats, see @rprogram{Invoking lilypond}.
+For more information on output formats, see @rprogram{lilypond}.
 
 @node Skipping corrected music
 @subsection Skipping corrected music
--- lilypond-2.10.0/scm/documentation-lib.scm.orig	2006-11-10 14:44:26.000000000 +0100
+++ lilypond-2.10.0/scm/documentation-lib.scm	2006-11-20 20:16:58.303967250 +0100
@@ -111,10 +111,10 @@
    "\\input texinfo @c -*-texinfo-*-"
    "\n@setfilename " file-name ".info"
    "\n@settitle " name
-   "\n@dircategory LilyPond"
+   "\n@dircategory LilyPond:"
    "\n@direntry"
    ;; prepend GNU for dir, must be unique
-   "\n* GNU " name ": (" file-name ").          " name "."
+   "\n* GNU " name ": (" file-name ").	" name
    "\n@end direntry\n"
    "@documentlanguage en\n"
    "@documentencoding utf-8\n"))
